tracer/test/Datadog.Trace.TestHelpers/TracerHelper.cs

```diff
@@ -15,14 +17,36 @@ internal static class TracerHelper
         /// <summary>
         /// Create a test instance of the Tracer, that doesn't used any shared instances
         /// </summary>
-        public static Tracer Create(
+        public static ScopedTracer Create(
             TracerSettings settings = null,
             IAgentWriter agentWriter = null,
             ITraceSampler sampler = null,
             IScopeManager scopeManager = null,
             IDogStatsd statsd = null)
         {
-            return new Tracer(settings, agentWriter, sampler, scopeManager, statsd);
+            return new ScopedTracer(settings, agentWriter, sampler, scopeManager, statsd);
+        }
+
+        /// <summary>
+        /// Create a test instance of the Tracer, that doesn't used any shared instances
+        /// </summary>
+        public static ScopedTracer CreateWithFakeAgent(
+            TracerSettings settings = null)
+        {
+            return new ScopedTracer(settings, Mock.Of<IAgentWriter>());
```

Comment:
I kinda feel like this should probably be the _default_ in the `TracerHelper` 🤔

Reply:
Honestly I'm 50/50 on this one. "agent disabled by default" is the best choice from a performance perspective, but "agent enabled by default" is the path of least surprise.